### ISSUE TITLE
Add traffic mirroring on the load balancer interface

### ISSUE THREAD

**Issue by thebsdbox** [kube-vip/kube-vip#589]
**Is your feature request related to a problem? Please describe.**
This will add the capability to mirror all inbound traffic to the interface (all VIPs) to a secondary interface

**Describe the solution you'd like**
N/A

**Describe alternatives you've considered**
N/A

**Additional context**
[Example code](https://gist.github.com/mcastelino/7d85f4164ffdaf48242f9281bb1d0f9b)

**Comment by lubronzhan** [comment]
Hi Dan, Is the main purpose of this to allow customer to run a process on the same host connecting second nic to monitor the traffic of LB?

**Comment by thebsdbox** [comment]
Yeah, the idea is to duplicate all traffic from the adapter A and send it down adapter B, where a scanning service would be sat processing the data.

**Comment by lubronzhan** [comment]
I was able to create a POC to fulfill that.

https://github.com/lubronzhan/port-mirror-testing/blob/main/pkg/mirror/mirror.go

I can pick it up.

Do we just want to mirror the traffic that goes through interface of service type LB, or also the traffic that goes through the control plane HA endpoint interface?

**Comment by thebsdbox** [comment]
Just looked at the code, this would be great!

**Comment by lubronzhan** [comment]
Cool let me create a PR for that
/assign
